crates/config/src/lib.rs

```diff
@@ -397,6 +401,7 @@ pub struct Config {
     /// If this limit is exceeded, a `MemoryLimitOOG` result is thrown.
     ///
     /// The default is 128MiB.
+    // todo(onbjerg): this seems unused.
     pub memory_limit: u64,
```

Comment:
revm does allow configuring this and we are passing the value into it; must not be wired correctly